lib/python/util.py

```diff
@@ -34,3 +34,21 @@ def list_files_recursively(dir, pattern):
             matches.append(os.path.join(root, filename))
 
     return matches
+
+def get_git_revision_script_url(file_path):
+    import os
+    import subprocess
+    """
+    file_path is the local file path from a git repo
+    returns the github url with the hash value of the current HEAD
+    """
+    curr_dir = os.getcwd()
+    os.chdir(os.path.dirname(file_path)) # need to chg dir to run git commands
+
+    repo_username_name = (subprocess.check_output(['git', 'config', '--get', 'remote.origin.url']).strip()).split(':')[1]
+    repo_name          = repo_username_name.split('/')[1]
+    hash_val           = subprocess.check_output(['git', 'rev-parse', 'HEAD']).strip()
+    script_rel_path    = file_path[file_path.index(repo_name) + len(repo_name) + 1:]
```

Comment:
actually the same command does't return the same thing for all machines/repos. looking at it now